Python/Product/PythonTools/PythonTools/LanguageServerClient/PythonLanguageClient.cs

```diff
@@ -593,24 +590,35 @@ private Tuple<StreamData, bool> OnSendToServer(StreamData data) {
             return Tuple.Create(data, true);
         }
 
-        private async Task OnWorkspaceOpening(object sende, EventArgs e) {
+        private IEnumerable<WorkspaceFolder> GetFolders() {
```

Comment:
Could you have this always return a list? Every place you call it you're turning it into a list.

Reply:
true. such a small list. no reason to make it enumerable